src/crypto/mbedtls/key_pair.cpp

```diff
@@ -286,192 +295,8 @@ namespace crypto
     registeredID = 8
   };
 
-  static inline int x509write_crt_set_subject_alt_name(
```

Comment:
This was unused and the sister `x509write_crt_set_subject_alt_names()` (plural) is also deleted.

Reply:
We could still set them, even if mbedTLS can't parse them back. Not worth the effort though, I think.